audformat/core/utils.py

```diff
@@ -474,32 +483,38 @@ def expand_file_path(
         index: pd.Index,
         root: str,
 ) -> pd.Index:
-    r"""Expand relative path in index with root.
+    r"""Expand path in index with root.
+
+    It applies :func:`os.path.normpath`
+    to the provided path ``root``,
+    adds a file separator at its end
+    and puts it in front of the file path in the index.
 
     Args:
-        index: index with relative file path conform to
+        index: index with absolute or relative file path
+            conform to
             :ref:`table specifications <data-tables:Tables>`
-        root: directory added in front of the relative file path
+        root: relative or absolute path added in front
+            of the index file path
 
     Returns:
-        index with absolute file path
+        index with root added to file path
 
     Raises:
         ValueError: if index is not conform to
             :ref:`table specifications <data-tables:Tables>`
 
     Examples:
-        >>> index = filewise_index(['f1', 'f2'])
-        >>> index
-        Index(['f1', 'f2'], dtype='string', name='file')
-        >>> expand_file_path(index, '/some/where')  # doctest: +SKIP
-        Index(['/some/where/f1', '/some/where/f2'], dtype='string', name='file')
+        >>> expand_file_path(filewise_index(['f1', 'f2']), '/a')
+        Index(['/a/f1', '/a/f2'], dtype='string', name='file')
+        >>> expand_file_path(filewise_index(['/f1', '/f2']), './a')
```

Comment:
I am not sure if we should include an example that results in an invalid path. Maybe instead add a note in the docstring that the result may contain invalid paths and we do not check for it.

Reply:
I don't think a path containing `//` is invalid. But I agree that it is maybe not a good idea to include such an example.